src/template/header.html.twig

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="fr">
+    <head>
```

Comment:
You may also move this <head> tag to a dedicated .html.twig file, like you did for footer.html.twig

Reply:
Done